include/lemon/matrix.hpp

```diff
@@ -0,0 +1,280 @@
+#ifndef LEMON_MATRIX_HPP
+#define LEMON_MATRIX_HPP
+
+#include <chemfiles/types.hpp>
+
+#include <array>
+#include <cmath>
+#include <complex>
+#include <vector>
+#include <algorithm>
+
+namespace lemon {
+
+using Vector3D = chemfiles::Vector3D;
+using Matrix3D = chemfiles::Matrix3D;
+using Coordinates = std::vector<Vector3D>;
+
+using chemfiles::cross;
+using chemfiles::dot;
+
+inline double trace(const Matrix3D& matrix) {
```

Comment:
This could be added to Matrix3D if you prefer

Reply:
Sure, I put it here in the interest of time - there's a couple of other things that I will probably add upstream as well!

Reply:
👍